### PR TITLE
[OCPTELAN-658] Update grafana url per namespace for metrics_analyzer

### PR DESCRIPTION
**Pull request by dvandra**
Using grafana dashboard uid as env var.

### CODE REVIEW

File: batch_worker.py [hunk 1]
```diff
@@ -140,6 +141,11 @@ def get_cluster_id_batch(self,end_date, batch_size):
         ]
         return result
     
+    def get_grafana_url_for_namespace(self, df):
+        return f"""{df['grafana_url']}/d/{self.grafana_dashboard_uid}/
```
Comment (xrajesh) [inline batch_worker.py:145]: @dvandra We could have a check for valid value for df['grafana_url']

File: input/thanos.json [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
-         },
-         "url": "https://rbac-query-proxy-open-cluster-management-observability.apps.ocp-412-4xworker-cgk4m.dev07.red-chesterfield.com",
-         "token": "sha256~"
-       },
-       {
-         "hub_name":"hub-2",
-         "url": "https://rbac-query-proxy-open-cluster-management-observability.apps.bu-demo.bu.red-chesterfield.com",
-         "token": "sha256~"
-       }
-     ]
```
Comment (xrajesh) [inline input/thanos.json:18]: @dvandra There is no change to this file I think . Some formatting changed .. can we  remove the changes to this file .

File: recommend.sh [hunk 3]
```diff
@@ -4,5 +4,6 @@ export METRIC_DAYS=10
 export RESOLUTION=20m
 export LOGROOT=./logs
 export BATCH_SIZE=4
+export GRAFANA_DASHBOARD_UID=""
```
Comment (xrajesh) [inline recommend.sh:7]: We have to have a read me update on this field with default values .

File: batch_worker.py (hunk 1 above)
Comment (dvandra) [reply batch_worker.py:145]: Added one check for uid that it is not null.

**Review by xrajesh — APPROVED** [review]

**Pull request merged**